build-tools/automation/azure-pipelines.yaml

```diff
@@ -188,7 +188,7 @@ stages:
       inputs:
         solution: $(System.DefaultWorkingDirectory)/xamarin-android/build-tools/create-packs/Microsoft.Android.Sdk.proj
         configuration: $(XA.Build.Configuration)
-        msbuildArguments: /t:CreateAllPacks /restore /bl:$(System.DefaultWorkingDirectory)/bin/Build$(XA.Build.Configuration)/create-all-packs.binlog
+        msbuildArguments: /t:CreateAllPacks /restore /bl:$(System.DefaultWorkingDirectory)/xamarin-android/bin/Build$(XA.Build.Configuration)/create-all-packs.binlog
```

Comment:
Why the insertion of `xamarin-android/`?

Reply:
This path was incorrect before. `$(DefaultWorkingDirectory)` changes based on the jobs usage of the multi-repo checkout feature, and in this case the extra `xamarin-android` is needed to make sure this log ends up in the right place.

Reply:
indeed. and i just confirmed with this change that the `xa-build-status` zip artifact now includes `create-all-packs.binlog`